doc/command_ref.rst

```diff
@@ -182,7 +182,10 @@ Options
     Disable the listed plugins specified by names or globs.
 
 ``--disablerepo=<repoid>``
-    Disable specific repositories by an id or a glob. This option is mutually exclusive with ``--repo``.
+    Temporarily disable active repositories for the purpose of the current dnf command.
+    Accepts an id, a comma-separated list of ids, or a glob of ids. This option can be
+    specified multiple times, but is mutually exclusive with ``--repo``.
+    To permanently disable active repositories, use `dnf config-manager --disable <repo id>`
```

Comment:
Unfortunately, `dnf config-manager` is actually not yet part of the dnf core, so we can't refer to it here, as it's entirely possible that isn't present at all.